ChangeLog

```diff
@@ -45,11 +45,14 @@
  * Deprecated ComposedCopula in favor of BlockIndependentCopula
  * Added Polygon::FillBetween static method to fill the surface between two curves
  * Deprecated Study.printLabels (see getLabels)
+ * Removed optional parameters from Contour constructors, use set methods or ResourceMap keys to set them
```

Comment:
@jschueller I believe I have implemented all your suggestions, but since I am not the owner of this PR, I cannot resolve conversations. Could you please do this?

Reply:
If the tests pass, this is ready to be merged IMO.